pcsx2-qt/Settings/AchievementSettingsWidget.cpp

```diff
@@ -70,10 +75,21 @@ AchievementSettingsWidget::AchievementSettingsWidget(SettingsDialog* dialog, QWi
 	dialog->registerWidgetHelp(m_ui.primedIndicators, tr("Show Challenge Indicators"), tr("Checked"),
 		tr("Shows icons in the lower-right corner of the screen when a challenge/primed achievement is active."));
 
+	{
+		const char* duration_title = "Notification Duration";
+		const char* duration_recommended_value = "5 seconds";
+		const char* duration_description = "The duration, in seconds, an achievement popup notification will remain on screen.";
+
+		dialog->registerWidgetHelp(m_ui.notifications_duration, tr(duration_title), tr(duration_recommended_value), tr(duration_description));
```

Comment:
This won't work for localization. The strings need to be inline, as part of the `tr()` expression.

Reply:
Good to know! I'll move the strings so they're inline with the `tr()` expressions